.github/fabricbot.json

```diff
@@ -1,1855 +0,0 @@
-            "parameters": {
-              "days": 7
-            }
-          },
-          {
-            "name": "noLabel",
-            "parameters": {
-              "label": "Status: No Recent Activity"
-            }
-          }
-        ],
-        "actions": [
-          {
-            "name": "addLabel",
-            "parameters": {
-              "label": "Status: No Recent Activity"
-            }
-          },
-          {
-            "name": "addReply",
-            "parameters": {
-              "comment": "This pull request has been automatically marked as stale because it has been marked as requiring author feedback but has not had any activity for **7 days**. It will be closed if no further activity occurs **within 7 days of this comment**."
-            }
-          }
-        ]
-      },
-      "disabled": true
-    },
-    {
-      "taskType": "trigger",
-      "capabilityId": "IssueResponder",
-      "subCapability": "IssuesOnlyResponder",
-      "version": "1.0",
-      "config": {
-        "conditions": {
-          "operator": "and",
-          "operands": [
-            {
-              "name": "labelAdded",
-              "parameters": {
-                "label": "Needs-CPS-work"
-              }
-            }
-          ]
-        },
-        "eventType": "issue",
-        "eventNames": [
-          "issues",
-          "project_card"
-        ],
-        "taskName": "Needs-CPS-Work",
-        "actions": [
-          {
-            "name": "addReply",
-            "parameters": {
-              "comment": "Please create an issue in AzDo to link to this one"
-            }
-          }
-        ]
-      },
-      "disabled": true
-    },
-    {
-      "taskType": "trigger",
-      "capabilityId": "IssueResponder",
-      "subCapability": "PullRequestResponder",
-      "version": "1.0",
-      "config": {
-        "conditions": {
-          "operator": "and",
-          "operands": [
-            {
-              "name": "prTargetsBranch",
-              "parameters": {
-                "branchName": "dev16.8.x"
-              }
-            },
-            {
-              "name": "isAction",
-              "parameters": {
-                "action": "merged"
-              }
-            }
-          ]
-        },
-        "eventType": "pull_request",
-        "eventNames": [
-          "pull_request",
-          "issues",
-          "project_card"
-        ],
-        "taskName": "Milestone tracking - 16.8",
-        "actions": [
-          {
-            "name": "addMilestone",
-            "parameters": {
-              "milestoneName": "16.8"
-            }
-          }
-        ]
-      }
-    },
-    {
-      "taskType": "trigger",
-      "capabilityId": "InPrLabel",
-      "subCapability": "InPrLabel",
-      "version": "1.0",
-      "config": {
-        "fixedLabelEnabled": true,
-        "label_fixed": "Resolution-Fixed",
-        "taskName": "Mark bugs with \"Resolution-Fixed\""
-      }
-    },
```

Comment:
@JohannesLampel The `InPrLabel` task didn't get ported over. I'm guessing because the config has `fixedLabelEnabled` set to `true`; I don't see that behavior in the Policy Service implementation though.

Please advise on this.

Reply:
@JohannesLampel followed up offline and stated that the 'fixed' behavior of the in-pr label feature is no longer supported.

@smitpatel -- Unfortunately, the new service cannot accomplish the behavior you had here, of marking the backing issue for a PR with a label when the PR is merged. We could label the issue while it's in PR, but that won't achieve the desired effect. If you want to gain this automation back, you'll need to explore authoring a GitHub Action to accomplish it.

Reply:
Thanks marking this as resolved.